static/js/zamboni/devhub.js

```diff
@@ -335,11 +335,11 @@ function addonFormSubmit() {
             parent_div.find(".item").removeClass("loaded").addClass("loading");
             var $document = $(document),
                 scrollBottom = $document.height() - $document.scrollTop(),
-                $form = $(this),
-                hasErrors = $form.find('.errorlist').length;
+                $form = $(this);
 
             $.post($form.attr('action'), $form.serialize(), function(d) {
                 parent_div.html(d).each(addonFormSubmit);
+                hasErrors = parent_div.find('.errorlist').length;
```

Comment:
Nice, well spotted. So, your change does fix the issue, but I think we should write all this more defensively. Can you try doing the following:
1. In the beginning of the function, do `$form = parent_div.find('form');`. Add a comment explaining that we need to do that because we just changed the whole html of the parent div.
2. When assigning `hasErrors` like you did, add a `var` to make sure it's not global and keep using the same assignation as before, `$form.find('.errorlist').length;`. Now that `$form` has been updated and we are in the function it should work, and be slightly more reliable

Reply:
Thanks for the hints. I just pushed a commit addressing your comments.